parser/src/python.lalrpop

```diff
@@ -1743,6 +1743,6 @@ extern {
         name => token::Tok::Name { name: <String> },
         "\n" => token::Tok::Newline,
         ";" => token::Tok::Semi,
-        "#" => token::Tok::Comment(_),
+        // "#" => token::Tok::Comment(_),
```

Comment:
hm, how does this work now?

Reply:
We already filter Comment before calling parser. So it was meaningless there for either feature flag on or off

Reply:
ah. right!